theme/themes/eea/elements/button.overrides

```diff
@@ -59,6 +51,19 @@
     background-color: @secondaryButtonColor;
     color: @secondaryButtonColorHover;
   }
+
+  &.icon.button i {
```

Comment:
any button should have the ability to add an icon not just the secondary, move this rule to the .ui.button basic style